modules/@apostrophecms/doc-type/ui/apos/components/AposDocEditor.vue

```diff
@@ -28,39 +28,44 @@
       />
       <AposButton
         type="primary" :label="saveLabel"
-        :disabled="docOtherFields.hasErrors || docUtilityFields.hasErrors"
+        :modifiers="buttonModifiers"
         @click="submit"
+        :tooltip="tooltip"
       />
     </template>
     <template #leftRail>
       <AposModalRail>
         <AposModalTabs
+          :key="tabKey"
```

Comment:
`key` trick here to force component refresh when needed because detecting changes in `errors` proved too deep